src/spec/product-type-importer.spec.coffee

```diff
@@ -0,0 +1,113 @@
+Promise = require 'bluebird'
+ProductTypeImporter = require '../lib/product-type-import'
+{SphereClient} = require 'sphere-node-sdk'
+
+errMissingCredentials = 'Missing configuration in env variables'
+
+argv =
+  clientId: process.env.SPHERE_CLIENT_ID
+  clientSecret: process.env.SPHERE_CLIENT_SECRET
+  projectKey: process.env.SPHERE_PROJECT_KEY
```

Comment:
But you don't run tests on iron.io. I'm just talking about tests here. Running locally like this it's not very helpful.